plugin/init.lua

```diff
@@ -79,7 +79,8 @@ local function split_nav(resize_or_move, key, direction)
     key = key,
     mods = modifier,
     action = wezterm.action_callback(function(win, pane)
-      if is_vim(pane) then
+      local num_panes = #win:window:active_tab():panes()
```

Comment:
Looks like this is tripping up the Selene linter: https://github.com/mrjones2014/smart-splits.nvim/actions/runs/14777660010/job/41489743641?pr=320

Reply:
You can try to fix the lint, or just add an ignore comment, I'm fine with either in this case, since this is very simple.

I'm not sure why it's getting tripped in the first place.

Reply:
Hey, sorry there was a typo, check now :)